### ISSUE TITLE
width: stretch not adding -webkit-fill-available

### ISSUE THREAD

**Issue by alexeymolchan** [postcss/autoprefixer#1441]
https://caniuse.com/?search=width%3A%20stretch

"autoprefixer": "^10.4.0",

- css input:

```
body {
  width: stretch;
}
```

- css output: 

```
body {
    width: -moz-available;
    width: stretch;
}
```

- expected output:

```
body {
    width: -moz-available;
    width: -webkit-fill-available;
    width: stretch;
}
```

- browserslist config:

```
[
    "last 2 Chrome versions",
    "last 2 Firefox versions",
    "last 2 Safari versions",
    "last 2 Edge versions",
    "last 2 ChromeAndroid versions",
    "last 2 iOS versions"
]
```
  

- autoprefixer info: 

```
Browsers:
  Chrome for Android: 96
  Chrome: 96, 95
  Edge: 96, 95
  Firefox: 94, 93
  iOS Safari: 15, 14.5-14.8
  Safari: 15.1, 15

These browsers account for 57.67% of all users globally

At-Rules:
  @resolution: webkit

Selectors:
  ::backdrop: webkit
  :autofill: webkit
  :fullscreen: webkit

Properties:
  appearance: webkit
  backdrop-filter: webkit
  backface-visibility: webkit
  background-clip: webkit
  box-decoration-break: webkit
  clip-path: webkit
  color-adjust: webkit
  hyphens: webkit
  mask-border-outset: webkit
  mask-border-repeat: webkit
  mask-border-slice: webkit
  mask-border-source: webkit
  mask-border-width: webkit
  mask-border: webkit
  mask-clip: webkit
  mask-composite: webkit
  mask-image: webkit
  mask-origin: webkit
  mask-position: webkit
  mask-repeat: webkit
  mask-size: webkit
  mask: webkit
  text-decoration-color: webkit
  text-decoration-line: webkit
  text-decoration-skip-ink: webkit
  text-decoration-skip: webkit
  text-decoration-style: webkit
  text-decoration: webkit
  text-emphasis-color: webkit
  text-emphasis-position: webkit
  text-emphasis-style: webkit
  text-emphasis: webkit
  text-size-adjust: webkit
  user-select: webkit

Values:
  cross-fade: webkit
  element: moz
  fill-available: webkit
  fill: webkit
  fit-content: moz
  image-set: webkit
  isolate: webkit
  stretch: moz

```

**Comment by ai** [comment]
I added better MDN data to `caniuse-lite` https://github.com/browserslist/caniuse-lite/commit/fc45fc1ab65aaac1678d279e673daa45f8c5c512

Now let’s wait for `caniuse-db` release because `caniuse-lite` versions are synced with `caniuse-db` and we can’t release `caniuse-lite` earlier.

**Comment by ai** [comment]
Fixed 09f1a89

**Comment by ai** [comment]
Released in 10.4.2.
